src/DGtal/geometry/helpers/ContourHelper.h

```diff
@@ -75,16 +76,45 @@ class ContourHelper
   /**
    * Checks if a contour given as a sequence of point is clockwise oriented or not.
    *
-   * @param aContour the vector containing the 2D contour coordinates.
+   * @param[in] aContour the vector containing the 2D contour coordinates.
    * @return 'true' if the contour is counter clockwise oriented, 'false' otherwise.
    **/
-
   template <typename TPoint> 
   static
   bool
   isCounterClockWise(const std::vector<TPoint> & aContour);
 
-   
+
+
+  /**
+   * @return the Freeman code associated to given points (0,1,2, or
+   * 3). If the direction between the two points differs from these
+   * directions it returns 8.
+   *
+   * @param[in] pt1  a first point.
+   * @param[in] pt2  a second point.
+   * 
+   **/
+  template<typename TPoint>
+  static 
+  unsigned int 
+  getFreemanCode4C(const TPoint &pt1, const TPoint &pt2);
+  
+
+  /**
+   * Transforms an input contour into an 8 connected contour.
+   * The contour is given from a range of points. 
```

Comment:
if the point sequence is not 8 connected, what happens?
This should be described here I think